source/_components/sensor.nut.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: NUT Sensor
+description: "Instructions on how to set up NUT sensors within Home Assistant."
+date: 2016-11-23
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nut.png
+ha_category: Sensor
+---
+
+The `nut` sensor platform allows you to monitor a UPS (battery backup) by using data from the NUT server.
```

Comment:
This should most likely be "a NUT server" since I assume there can potentionaly be more than one host.